homework9/hw9task3/iterators_3/universal_file_counter.py

```diff
@@ -20,7 +20,7 @@ def universal_file_counter(
     amount_of_tokens_or_lines = 0
     all_files = [file for file in os.listdir(dir_path) if file.endswith(file_extension)]
     for file_name in all_files:
-        file_path = str(dir_path) + "\\" + file_name
+        file_path = str(dir_path) + "/" + file_name
```

Comment:
Thank you for pointing this out. Please do research on how to join paths in a cross-platform way.

Reply:
Oh, yes, we can do like `"/".join((dir_path,file_name))` or like f"{dir_path}/{file_name}"

Reply:
Is it `cross-platform`?

Reply:
No, it is a common way to join smth. Cross-platform ones are:
1) using **os.path.join**, like `file_path = os.path.join(dir_path, file_name)`;
2) using **Path()** with operator **/**, like `data_folder = Path("source_data/text_files/")`
    `file_to_open = data_folder **/** "raw_data.txt"`
One of good benefits using this way is that we can read the contents of a text file without having to mess with opening and closing the file using `file_to_open.read_text()`.
As I understand these are the most common ways to join paths in a cross-platform way.